src/styles.scss

```diff
@@ -23,6 +23,10 @@ td.govuk-table__cell {
   white-space: pre-wrap;
 }
 
+td.govuk-table__cell a.govuk-link {
+  float: right;
```

Comment:
Pity there's not a pull-right or rfloat class to defined in styles.scss

Reply:
There wasn't anything there already to use so I added this.

Reply:
If we need any more float rights we should consider adding a generic style in future.